### PR TITLE
refactor: create enums for strings in dataclasses

### PR DESCRIPTION
**Pull request by Jackmin801**
Goals:
- Address https://github.com/jina-ai/docarray/issues/449

### CODE REVIEW

File: docarray/dataclasses/getter.py [hunk 1]
```diff
@@ -1,47 +1,48 @@
...
     return doc.uri or doc.tensor
 
 
-def tabular_getter(doc: 'Document'):
+def tabular_getter(doc: "Document"):
     return doc.uri
 
 
-def blob_getter(doc: 'Document'):
+def blob_getter(doc: "Document"):
```
Comment (JoanFM) [inline docarray/dataclasses/getter.py:43]: keep single quote, we have a prehook applying style changes no need to apply ur own

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: docarray/dataclasses/getter.py (hunk 1 above)
Comment (Jackmin801) [reply docarray/dataclasses/getter.py]: ok ive taken out the style commits

File: docarray/dataclasses/types.py [hunk 2]
```diff
@@ -21,6 +21,7 @@
 
 from docarray.dataclasses.getter import *
 from docarray.dataclasses.setter import *
+from docarray.dataclasses.enums import *
```
Comment (alaeddine-13) [inline docarray/dataclasses/types.py:24]: why do we need to expose these ?

Comment (Jackmin801) [reply docarray/dataclasses/types.py:24]: Because AttributeType and DocumentMetadata are used in this file

Comment (alaeddine-13) [reply docarray/dataclasses/types.py:24]: ah nvm thought all this is imported in docarray/dataclasses/__init__.py

**Review by alaeddine-13 — APPROVED** [review]

**Review by JoanFM — APPROVED** [review]

**Pull request merged**